.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "presets": ["env"],
+  "plugins": [
+        "transform-decorators-legacy",
```

Comment:
I'm slightly worried about this dep. It's required for using decorators, but when you don't have it the warning seems to make it sound like they aren't really supported yet. `legacy` makes it feel like the support will go away?

Reply:
My assumption here was that this plugin is just for this project; not node projects in general. If the intention is to automatically include this in all new node projects, then, yeah, it should be removed

Reply:
The same goes for all of the other babel plugins. Only include what is needed for the project. the `env` preset is a good default.

Reply:
Agreed. I'm removing both `transform-decorators-legacy` and `transform-exponentiation-operator` because I don't think we'll need them.